net/common.sh

```diff
@@ -99,14 +99,15 @@ SOLANA_CONFIG_DIR=$SOLANA_ROOT/config
 # Clear the current cluster configuration
 clear_config_dir() {
   declare config_dir="$1"
+
+  setup_secondary_mount
```

Comment:
@t-nelson FYI here, we need to make sure the secondary is linked before clearing, otherwise we might not actually delete the real contents in the unlinked config dir.